pkg/pool-weighted/contracts/test/MockManagedPoolSwapFeesLib.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "../managed/ManagedPoolSwapFeesLib.sol";
+
+contract MockManagedPoolSwapFeesLib {
+    bytes32 private _poolState;
+
+    // Helpers to decode Pool state
+
+    function getSwapFeePercentage() external view returns (uint256) {
+        return ManagedPoolStorageLib.getSwapFeePercentage(_poolState);
+    }
```

Comment:
Wouldn't it be better to pass and receive the state, instead of having it be on storage? Not a huge deal either way

Reply:
This is a hack so that we don't need to be smart about decoding it in the tests. This will go away with #1776